### ISSUE TITLE
Add integration between Substrate.lib and Transcalc tool

### ISSUE THREAD

**Issue by ra3xdh** [ra3xdh/qucs_s#354]
Transcalc tool requires substrate properties to perform the line calculation. The current implementation requires hand input of these properties. It's need to add a possibility to select the substrate material from the drop-down list. The material properties should be taken from `Substrate.lib` library.

**Comment by tomhajjar** [comment]
As long as manual entry is maintained it will be a useful feature.

Generic FR-4 Er varies depending on manufacturer and frequency of operation.

**Comment by ra3xdh** [comment]
Yes, the possibility to select the substrate material will be an addition to the manual input.

**Comment by ra3xdh** [comment]
Implemented via #1722
